lti_consumer/lti_xblock.py

```diff
@@ -1085,6 +1085,15 @@ def lti_1p3_launch_callback(self, request, suffix=''):  # pylint: disable=unused
             # This is optional though
             lti_consumer.set_launch_presentation_claim('iframe')
 
+            # Set context claim
+            # This is optional
+            lti_consumer.set_context_claim(
+                self.context_id,
+                context_types=["course_section"],
+                context_title=self.course.display_name_with_default,
+                context_label=self.course.display_org_with_default
```

Comment:
I think that to better match the spec (http://www.imsglobal.org/spec/lti/v1p3/#context-claim) here we should use:

- **label**: course id (same as the `id` parameter)
- **title**: full course name + organization.

Reply:
Done in next commit